### PR TITLE
Select cell functions

### PR DESCRIPTION
**Pull request by joneit**
There is no ticket associated with this PR.

Fixed vertical scroll max calculation (was not displaying final rows).

This in turn fixed `selectFinalCell` and `selectToFinalCell` which failed because final row was not displayable.

Also refined calculation of column with for handle column which was too narrow to accommodate row numbers when bolded (row selected) and was showing elipsis.

### CODE REVIEW

**Comment by Dwaynekj** [comment]
This brings up questions I have about [this](https://appoji.jira.com/browse/GRID-331). Since subgrids are arbitrary is this a matter of just customizing the scroll bar dom element size

**Comment by joneit** [comment]
The scrollbars can be custom-positioned.

**Comment by joneit** [comment]
@Dwaynekj Ready for final review on this. Let's get this sucker merged!

**Comment by Dwaynekj** [comment]
Manual testing looks good

File: src/Hypergrid.js [hunk 1]
```diff
@@ -2344,5 +2344,7 @@ Hypergrid.localization = {
 Hypergrid.prototype.mixIn(require('./lib/events'));
 Hypergrid.prototype.mixIn(require('./lib/selection'));
 
+Hypergrid.properties = globalProperties;
```
Comment (joneit) [inline src/Hypergrid.js]: This is just defaults.js, which is the "shared" global properties object. Alterations here affect all grids, including simple overrides and theming. `grid.properties` is extended (created) from this and these are the only base layers now (down from 5 or 6 layers). Additions to `grid.properties` override the global props.

**Comment by joneit** [comment]
@Dwaynekj Did you notice the new "Link styles" dashboard control?

File: src/defaults.js [hunk 2]
```diff
@@ -255,6 +262,13 @@ var defaults = {
 
     /**
      * @default
+     * @type {string}
+     * @memberOf module:defaults
+     */
+    filterRenderer: 'SimpleCell',
```
Comment (Dwaynekj) [inline src/defaults.js]: Why was this added?
